src/utils.js

```diff
@@ -15,11 +15,11 @@ function createAssetsFilter(excludePatterns) {
         pattern = new RegExp(pattern, 'u');
       }
 
-      if (_.isRegExp(pattern)) {
+      if (types.isRegExp(pattern)) {
```

Comment:
What is `types`?

Reply:
New node.js utilities
https://nodejs.org/api/util.html#util_util_types

Reply:
Nice, didn't know about it.